ios/RNMagicScript/components/UiNodeSelector.swift

```diff
@@ -61,3 +61,5 @@ import SceneKit
         return node as? Dragging
     }
 }
+
+extension UiNodeSelector: UiNodeSelecting { }
```

Comment:
Do we need extension here? Can we declare protocol in line 20 - similar to UiNodeAnimator?

Reply:
Sure!